### PR TITLE
mpg321: add page

### PR DESCRIPTION
**Pull request by temuraru**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/mpg321.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# mpg321
+
+> High Performance MPEG 1.0/2.0/2.5 Audio Player for Layer 1, 2, and 3.
+> mpg321 was written to be a drop-in replacement for the (by-then) non-free mpg123 player.
```
Comment (zdroid) [inline pages/common/mpg321.md:4]: Is this line needed? Feels like it isn't really required.

File: pages/common/mpg321.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Play an audio source (on repeat) exactly N=3 times:
```
Comment (zdroid) [inline pages/common/mpg321.md:6]:
```suggestion
- Play an audio source 3 times on repeat:
```

File: pages/common/mpg321.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Play files randomly until interrupted:
+
+`mpg321 -Z {{ file(s) | URL }}`
+
+- Shuffle list of files before playing the list once:
+
+`mpg321 -z {{ file(s) | URL }}`
+
+- Play all the files in the current folder(s) and subfolders, randomly (until interrupted), with Basic keys enabled:
```
Comment (zdroid) [inline pages/common/mpg321.md:30]:
```suggestion
- Play all the files in the current directory, randomly (until interrupted), with Basic keys enabled:
```

File: pages/common/mpg321.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- Play an audio source (on repeat) exactly N=3 times:
+
+`mpg321 -l {{3}} {{ file(s) | URL }}`
+
+- Play an audio source (on repeat) forever (0 means until interrupted):
+
+`mpg321 -l 0 {{ file(s) | URL }}`
+
+- Play a whole folder (recursively):
```
Comment (zdroid) [inline pages/common/mpg321.md:14]:
```suggestion
- Play a whole directory recursively:
```

File: pages/common/mpg321.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+# mpg321
+
+> High Performance MPEG 1.0/2.0/2.5 Audio Player for Layer 1, 2, and 3.
+> mpg321 was written to be a drop-in replacement for the (by-then) non-free mpg123 player.
+
+- Play an audio source (on repeat) exactly N=3 times:
+
+`mpg321 -l {{3}} {{ file(s) | URL }}`
```
Comment (zdroid) [inline pages/common/mpg321.md:8]: `{{ file(s) | URL }}` should be something like `{{path/to/file|url}}` and folder should be replaced with directory in other examples. Perhaps other maintainers have a better idea how to handle this.

**Comment by sbrl** [comment]
Hey! Is there any reason why you closed this PR? We'd loe to help you out if you're encountering difficulties.

Also, next time perhaps you could avoid force-pushing please? It makes it harder to tell what's changed when reviewing, and we've got a special button that allows us to squash easily on merge :smiley_cat:

**Pull request closed without merging**